reconcile/utils/terraform_client.py

```diff
@@ -716,6 +716,22 @@ def _can_skip_rds_modifications(
                 response = self._aws_api.describe_rds_db_instance(
                     account_name, resource_name, region_name=region_name
                 )
+                replica_identifiers = response["DBInstances"][0].get(
```

Comment:
I don't think validations should be made in `_can_skip_rds_modifications`, they should be made in